notifique/src/main/java/com/nathanrassi/notifique/NotifiqueApplication.java

```diff
@@ -26,4 +26,4 @@
   @Override public AndroidInjector<Service> serviceInjector() {
     return serviceInjector;
   }
-}
+}
```

Comment:
done. I finally understood what you're saying. I have literally no idea why this is necessary, but whatever :p